exercises/series/uSeriesTest.pas

```diff
@@ -0,0 +1,189 @@
+unit uSeriesTest;
+
+interface
+uses
+  DUnitX.TestFramework, uSeries;
+
+const
+  CanonicalVersion = '1.0.0';
+
+type
+  [TestFixture]
+  TSeriesTest = class(TObject)
+  private
+    procedure CompareArrays(Array1, Array2: TArray<string>);
+  public
+    [Test]
+//    [Ignore('Comment the "[Ignore]" statement to run the test')]
+    procedure slices_of_one_from_one;
+
+    [Test]
+    [Ignore]
+    procedure slices_of_one_from_two;
+
+    [Test]
+    [Ignore]
+    procedure slices_of_two;
+
+    [Test]
+    [Ignore]
+    procedure slices_of_two_overlap;
+
+    [Test]
+    [Ignore]
+    procedure slices_can_include_duplicates;
+
+    [Test]
+    [Ignore]
+    procedure slices_of_a_long_series;
+
+    [Test]
+    [Ignore]
+    procedure slice_length_is_too_large;
+
+    [Test]
+    [Ignore]
+    procedure slice_length_cannot_be_zero;
+
+    [Test]
+    [Ignore]
+    procedure slice_length_cannot_be_negative;
+
+    [Test]
+    [Ignore]
+    procedure empty_series_is_invalid;
+  end;
+
+implementation
+
+uses System.SysUtils;
+
+procedure TSeriesTest.CompareArrays(Array1, Array2: TArray<string>);
+var
+  i: integer;
+begin
+  Assert.AreEqual(Length(Array1), Length(Array2), ' - Arrays doesn''t have same length');
+  for i := Low(Array1) to High(Array1) do
+    Assert.AreEqual(Array1[i], Array2[i], format(' - on [%d] place of array', [i]));
+end;
+
+procedure TSeriesTest.empty_series_is_invalid;
+var
+  Actual : TSlice;
+begin
+  Actual := TSlice.Create('');
+  Assert.WillRaiseWithMessage(procedure()
```

Comment:
The `()` are not wrong after `procedure`, but they are also not needed.

Reply:
Yes, you are right. I have changed it.